docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+db_action () {
+  while [[ $(pg_isready -U "${DB_USER}" -h "${DB_HOST}" -d "${DB_NAME}") == *"no response"* ]]
+  do
+    echo "Database is not ready yet."
+    sleep 5s
+  done
+
+  pg_isready -U "${DB_USER}" -h "${DB_HOST}" -d "${DB_NAME}" && echo "Database is ready."
+
+  # changeLogFile must be relative path
+  DB_UPDATE=$(liquibase --changeLogFile=../../../${CHANGELOG_FILE} --hub-mode=off --username="${DB_USER}" --password="${DB_PASSWORD}" --url=jdbc:postgresql://"${DB_HOST}":"${DB_PORT}"/"${DB_NAME}" "$1")
+
+  if $DB_UPDATE; then
+    echo "Success"
+  else
+    echo "Liquibase update failed"
+    exit 1
+  fi
```

Comment:
Doesn't the `liquibase` command return non-`0` in case of failure?

Reply:
I need to find a better way to catch the error/status code for this part since regular one like parsing `$?` after command doesn't work as expected

Reply:
hmm weird 🤔 — I think I did try running the `liquibase` command when I commented on this, and it seemed to return non-`0` when it failed (thus enabling parsing `$?`)

could you please have a second look at it?